src/markdown-pages/contribute-to-quickstarts/build-a-quickstart/contribute-quickstart.mdx

```diff
@@ -0,0 +1,213 @@
+
+```
+
+</>
+
+</Step>
+
+Here, you added slug, title and description for people to recognize and understand the purpose of your quickstart. You also refer the associated dashboards and alerts to create the quickstart. 
+
+<Step>
+
+Next, add a logo for your quickstart. It can be either ".svg" or ".png". Your quickstart directory should look similar to the following.
+
+![flashDB quickstart directory](../../../images/contribute-to-quickstarts/flashdb-quickstart-directory.png)
+
+</Step>
+
+<Step>
+
+Your quickstart is now ready to be published. You're going to commit your changes back to GitHub where it will be reviewed by New Relic.
+Follow the [conventional commit syntax](https://github.com/newrelic/developer-website/blob/main/CONTRIBUTING.md#using-conventional-commits) for New Relic to commit your changes.
+
+<>
+
+```bash
+git add -A
+git commit -m "feat(flashdb): Added a quickstart for flashdb"
+```
+
+</>
+
+</Step>
+
+<Callout variant="important">
+
+We, at New Relic, are always happy to receive contributions from the community. However, we are expecting a number of contributions for flashdb following the same course.
+To help us manage your contributions efficiently, we encourage you to label your contribution following this particular course as **quickstart-course** using `git tag` command.
+
+</Callout>
+
+<Step>
+
+Push your changes to GitHub:
+
+<>
+
+```bash
+git tag quickstart-course
+git push
+```
+</>
+
+</Step>
+
+</Steps>
+
+### Create a PR to contribute your quickstart
+
+Once you've pushed your changes to GitHub, you can [create a pull request](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request).
+
+<Steps>
+
+<Step>
+
+Navigate to your quickstart repo, click **Contribute** and then **Open pull request**.
+
+![pull request](../../../images/contribute-to-quickstarts/create-pull-request.png)
+
+</Step>
+
+<Step>
+
+This takes you to **Open a pull request page**. Here, add a summary for your quickstart and click **Create pull request**.
+
+![create pull request](../../../images/contribute-to-quickstarts/create-pr.png)
+
+</Step>
+
+<Callout variant='important'>
+
+When contributing a quickstart, make sure to go thorugh pre-checks.
+
+</Callout>
+
+Submit the PR and wait for the review. Once your quickstart passes all the checks, you can preview your quickstart on New Relic I/O.
+
+![quickstart preview link](../../../images/contribute-to-quickstarts/preview-link.png)
+
+This takes you to your quickstart on New Relic I/O.
+
+![flashdb quickstart preview](../../../images/contribute-to-quickstarts/flashdb-preview.png)
+
+<Callout variant="important">
+
+At this stage, your quickstart will be automatically validated in GitHub. If this were a real-life submission, it would then be approved and incorporated into New Relic I/O for your users to see. As this is a a demo course, we will not actually review this PR. Feel free to delete it after it validates your work.
```

Comment:
Just a thought, if you are encouraging them to raise a PR with their quickstart, maybe we could just ask them to raise it as a 'draft' PR instead of regular so it doesn't end up in our reviewers queue.